cartographer_ros/cartographer_ros/color.cc

```diff
@@ -14,20 +14,21 @@
  * limitations under the License.
  */
 
-#include "cartographer_ros/color_manager.h"
+#include "cartographer_ros/color.h"
 
 #include <cmath>
 
 #include "glog/logging.h"
 
-namespace cartographer_ros {
+namespace {
```

Comment:
Oh wait, sorry, I misunderstood. I will move the anonymous namespace inside cartographer_ros.